### ISSUE TITLE
Restructure Known Source and Plugins for Improved Maintenance and Development

### ISSUE THREAD

**Issue by lindner** [idno/Known#3129]
Today Known has a core repo in idno/known and a number of core plugins that were split off in the past into separate repos.

While this offers great flexibility it also makes developing Known difficult and requires a developer to juggle multiple repos, composer configs etc.  The plugin repos have separate issue trackers, are a chore to maintain, and require setting up configs across many places.

To solve this

- Import plugins into the main Known Repo (a list will be added as a comment).  For each plugin:
  - Import the code into an appropriate directory, preserve change history if possible.
  - Issues should also be moved to the main repo.
  - If there are composer.json for dependencies include them in the main composer.json.

Once this is done it will be much easier to make sure that all core plugins are functioning well.
